lib/jsdom/living/generator/events/EventTarget.idl

```diff
@@ -0,0 +1,6 @@
+[Exposed=(Window,Worker)]
```

Comment:
Heh, I just went by https://github.com/tmpvar/jsdom/pull/1227#discussion_r39228742 on this one.